docs/UserGuide.md

```diff
@@ -281,14 +284,44 @@ If your changes to the data file make its format invalid, FoodWhere will discard
-| **rsort** | `rsort CRITERION` <br> e.g., `rsort rating`                                                                  |                                                                   
-| **sadd**  | `sadd n/NAME a/ADDRESS [t/TAGS]…` <br> e.g., `sadd n/John Chicken Rice a/Blk 123 Bedok South t/veryNice`     |
-| **sdel**  | `sdel INDEX`                                                                                                 |
-| **sedit** | `sedit INDEX [n/NAME] [a/ADDRESS] [t/TAGS]…` <br> e.g., `sedit n/John Chicken Rice a/Blk 123 Bedok South`    |
+| **rfind** | `rfind n/KEYWORD [KEYWORD]… t/KEYWORD [KEYWORD]…` <br> e.g., `rfind n/eatery`                                |
+| **rsort** | `rsort CRITERION` <br> e.g., `rsort rating`                                                                  |
+| **sadd**  | `sadd n/NAME a/ADDRESS [t/TAGS]…` <br> e.g., `sadd n/John Chicken Rice a/Blk 123 Bedok South`                |
+| **sdel**  | `sdel STALL_INDEX`                                                                                           |
+| **sedit** | `sedit STALL_INDEX [n/NAME] [a/ADDRESS] [t/EDIT_TAG]…` <br> e.g., `sedit 1 n/John Chicken Rice`                  |
 | **slist** | `slist`                                                                                                      |
-| **sfind** | `sfind n/NAME_KEYWORD [MORE_KEYWORDS]… t/TAG_KEYWORD [MORE_KEYWORDS]…` <br> e.g., `rfind n/eatery`           |
+| **sfind** | `sfind n/KEYWORD [KEYWORD]… t/KEYWORD [KEYWORD]…` <br> e.g., `rfind n/eatery`                                |
 | **ssort** | `ssort CRITERION` <br> e.g., `ssort reversedname`                                                            |
+
+## Accepted command arguments
+
+| Argument       | Format                                                                                                      |
+|----------------|-------------------------------------------------------------------------------------------------------------|
+| `ADDRESS`      | Any ASCII text, case insensitive                                                                            |
+| `CONTENT`      | Any ASCII text                                                                                              |
+| `CRITERION`    | Valid criteria are stated at the [sfind](#finding-a-stall-sfind), [rfind](#finding-a-review-rfind) commands |
+| `DATE`         | A date in the format DD/MM/YYYY, D/MM/YYYY, DD/M/YYYY, D/M/YYYY, or with dashes instead of slashes          |
+| `EDIT_TAG`     | A possibly empty alphanumeric token without spaces, case insensitive                                        |
+| `KEYWORD`      | An alphanumeric token without spaces, case insensitive                                                      |
+| `NAME`         | A nonempty alphanumeric string with spaces, case insensitive                                                |
```

Comment:
Resolves #297. Just a question do you want to change NAME to STALL_NAME?

Reply:
All names are the same, the name in review reflects the name of the stall, the address in review reflects the address of the stall.

Reply:
Alright noted. Since you split the index into STALL_INDEX and REVIEW_INDEX, thought we could do something similar for NAME but what you say made sense. This conversation will be resolved with no further follow-up action.